src/canmatrix/canmatrix.py

```diff
@@ -710,9 +708,16 @@ def calcDLC(self):
 
     def findNotUsedBits(self):
         """
-        Find unused bits in frame
+        Find unused bits in frame.
+
+        Represents the bit usage in the frame by means of a list with 64 items.
+        Every item represents one bit and contains index of signal, occupying that bit.
```

Comment:
Sadly, it simply doesn't contain 'the index' of the signal, it contains the index plus 1...  But, this is a code bug from my perspective.  #227  I suppose that in the mean time the docs should represent the actual functionality.